contracts/Vault.vy

```diff
@@ -1260,7 +1260,7 @@ def erc20_safe_transfer(_token: address, _to: address, _value: uint256):
 
 
 @external
-def sweep(_token: address):
+def sweep(_token: address, _value: uint256):
```

Comment:
Would be neat to do the default value of `MAX_UINT256` (which just does balanceOf)